Assets/HoloToolkit/Input/Scripts/GestureManager.cs

```diff
@@ -251,5 +272,26 @@ void OnDestroy()
             InteractionManager.SourceUpdated -= InteractionManager_SourceUpdated;
             InteractionManager.SourceLost -= InteractionManager_SourceLost;
         }
+
+        public void OnSelect()
+        {
+            ThrowDeprecatedWarning();
```

Comment:
As explained somewhere else it's too early to add those warnings.. I don't want to have to comment those lines once I get the latest HTK (once/if this is merged)
Those warnings should be added once releaseNotes have been put in place